frontend/test/e2e/Tutorial.spec.ts

```diff
@@ -0,0 +1,328 @@
+import { WebDriver, By, until } from 'selenium-webdriver'
+
+import { ContractsPage } from '../pages/ContractsPage.js'
+import { before, describe, after, it } from 'node:test'
+import { expect } from 'chai'
+import { getDriver } from '../utils/driver.js'
+
+let driver: WebDriver
+let contractsPage: ContractsPage
+
+describe('Tutorial - Run all tutorial steps', () => {
+  before(async () => {
+    driver = await getDriver()
+    await driver.manage().setTimeouts({ implicit: 10000 })
+    contractsPage = new ContractsPage(driver)
+  })
+
+  it('Should show the welcome step and navigate to the next', async () => {
+    await contractsPage.navigate()
+    await contractsPage.waitUntilVisible()
+
+    const welcomeStepElement = await driver.wait(
```

Comment:
@bradleySuira, could you please wrap these driver calls so they can be reused easily?

Reply:
done, added page utils and utilize func to validate all steps